extra/utils/validation.js

```diff
@@ -0,0 +1,59 @@
+const users = [{
+    traineeEmail: 'trainee1@successive.tech',
+    reviewerEmail: 'reviewer1@successive.tech',
+    },
+    {
+        traineeEmail:'urvashi.chaudhary@successive.tech',
+        reviewerEmail:'mohammad.adil@successive.tech',
+    },
+    {
+        traineeEmail:'i@gmail.com',
+        reviewerEmail:'jfjjjjf',
+    },
+];
+
+
+function validateEmail(input){
+    let correct = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
```

Comment:
Regex should validate the email of specific domain such as (successive.tech).
e.g    trainee1@successive.tech --> valid
         user@successive.tech       --> valid
         trainee@gmail.com            --> invalid

Reply:
resolved